### PR TITLE
DOCS-629 Chef docs

### PR DESCRIPTION
**Pull request by ruthnaebeck**
### What does this PR do?
* Audit README
* Move development content into CONTRIBUTING.md
* Add CODEOWNERS

### Motivation
OKR - To have the README pull into the public docs site.

### Preview link
https://docs-staging.datadoghq.com/ruth/docs-629/agent/basic_agent_usage/chef

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,55 +1,127 @@
...
+
+There is a CouchDB recipe described in the `kitchen.yml`, and an Ubuntu platform with the 16.04 version.
+
+### Development loop
+
+The simplest way to develop fixes or features is to set up the platform and version of your choice with the `create` command and apply the recipe with the `converge` command. To explore the machine and test different things, login with the `login` command.
+
+**Note**: The `login` command only works on Linux and OSX. For Windows, connect to the VM through the Virtual Box interface or with putty or a similar ssh client.
+
+N.B.: The credentials of the created virtual machines are login `vagrant`, password `vagrant`.
```
Comment (apigirl) [inline CONTRIBUTING.md]:
```suggestion
**Note**: The credentials of the created virtual machines are login `vagrant`, password `vagrant`.
```

File: README.md [hunk 2]
```diff
@@ -132,250 +121,206 @@ E.g.
...
+2. Include the recipe in the `install-lifecycle` recipe:
+  ```ruby
+  include_recipe 'datadog::dd-agent'
+  ```
 
-### Agent v5 transitions
+### Integrations
 
-#### Upgrade from Agent v6 to Agent v7
+Enable Agent integrations by including the [recipe](#recipes) and configuration details in your role’s run-list and attributes. **Note**: You can create additional integration recipes by using the [datadog_monitor](#datadog-monitor) resource.
```
Comment (apigirl) [inline README.md]:
```suggestion
Enable Agent integrations by including the [recipe](#recipes) and configuration details in your role’s run-list and attributes. 
**Note**: You can create additional integration recipes by using the [datadog_monitor](#datadog-monitor) resource.
```

File: README.md [hunk 3]
```diff
@@ -1,127 +1,116 @@
-Where to Find the Code
-======================
-To submit issues and patches please visit https://github.com/DataDog/chef-datadog.
-The code is licensed under the Apache License 2.0 (see  LICENSE for details).
+# Datadog Chef Cookbook
 
-[![Chef cookbook](https://img.shields.io/cookbook/v/datadog.svg?style=flat)](https://github.com/DataDog/chef-datadog)
```
Comment (kbogtob) [inline README.md]: Why remove the badges?

Comment (ruthnaebeck) [reply README.md]: These were moved to CONTRIBUTING.md so they don't show on the docs site.

**Comment by kbogtob** [comment]
LGTM!

File: README.md [hunk 4]
```diff
@@ -1,127 +1,116 @@
...
-[![GitHub forks](https://img.shields.io/github/forks/DataDog/chef-datadog.svg)](https://github.com/DataDog/chef-datadog/network)
-[![GitHub stars](https://img.shields.io/github/stars/DataDog/chef-datadog.svg)](https://github.com/DataDog/chef-datadog/stargazers)
+The Datadog Chef recipes are used to deploy Datadog's components and configuration automatically. The cookbook includes support for:
 
-Datadog Cookbook
-================
+* Datadog Agent v7.x (default)
+* Datadog Agent v6.x
+* Datadog Agent v5.x
+* Log collection with Agent v6 & v7 (enable in [default.rb][1])
```
Comment (albertvaka) [inline README.md:8]:
```suggestion
```
I don't think we need to mention logs specifically (the same way we don't other Agent features like APM).

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,55 +1,127 @@
...
+* If you think you've found an issue, search the issue list to see if there's an existing issue.
+* Then, if you find nothing, open a Github issue.
 
 ## Pull Requests
 
 Have you fixed a bug or written a new feature and want to share it? Many thanks!
 
-In order to ease/speed up our review, here are some items you can check/improve
-when submitting your PR:
+In order to ease/speed up our review, here are some items you can check/improve when submitting your PR:
```
Comment (cswatt) [inline CONTRIBUTING.md:23]:
```suggestion
When submitting your PR, here are some items you can check or improve to facilitate the review process:
```

File: README.md [hunk 6]
```diff
@@ -132,250 +120,207 @@ E.g.
...
+| Property                   | Description                                                                                                                                                                                                                                                                                    |
+|----------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `'name'`                   | The name of the Agent integration to configure and enable                                                                                                                                                                                                                                      |
```
Comment (cswatt) [inline README.md:313]:
```suggestion
| `'name'`                   | The name of the Agent integration to configure and enable.                                                                                                                                                                                                                                      |
```

File: README.md [hunk 7]
```diff
@@ -132,250 +120,207 @@ E.g.
...
+| `use_integration_template` | Set to `true` (recommended) to use the default template, which writes the values of `instances`, `init_config`, and `logs` in the YAML under their respective keys. This defaults to `false` for backward compatibility, but will default to `true` in a future major version of the cookbook. |
 
-### Example
+#### Example
 
-This example enables the ElasticSearch integration by using the `datadog_monitor` resource. It provides the instance configuration (in this case: the url to connect to ElasticSearch) and sets the `use_integration_template` flag to use the default configuration template. Also, it notifies the `service[datadog-agent]` resource in order to restart the Agent.
+This example enables the ElasticSearch integration by using the `datadog_monitor` resource. It provides the instance configuration (in this case: the url to connect to ElasticSearch) and sets the `use_integration_template` flag to use the default configuration template. Also, it notifies the `service[datadog-agent]` resource to restart the Agent.
```
Comment (cswatt) [inline README.md:321]:
```suggestion
This example enables the ElasticSearch integration by using the `datadog_monitor` resource. It provides the instance configuration (in this case: the URL to connect to ElasticSearch) and sets the `use_integration_template` flag to use the default configuration template. Also, it notifies the `service[datadog-agent]` resource to restart the Agent.
```

**Pull request merged**